packages/docs/docs/docs/deploying.md

```diff
@@ -115,10 +115,14 @@ And back in the original terminal:
 zos push --network local
 ```
 
-This creates a `zos.dev-<network_id>.json` file with all the information about your
-project in this specific network. You can read more about this file format
+This deploys `MyContract` to the specified network. If your project added other contracts (using the `add` command) they would be deployed as well. An important thing to understand is that the contracts deployed by the `push` command are implementation or logic contracts and are not intended to be used directly, rather to be linked to a proxy as we will see later in the [Upgrading your project](https://docs.zeppelinos.org/docs/upgrading.html) section.
+
+Calling `push` creates a `zos.dev-<network_id>.json` file with all the information about your
```

Comment:
```suggestion
Calling `push` creates a `zos.network.json` file with all the information about your
```

`dev-network_id` is only used when the network is not well-known, such as kovan or mainnet

Reply:
I think that's reasonable, because the narrative is using `network --local` at this point. It later mentions a "Deploying to mainnet" guide.